docker/README.md

```diff
@@ -0,0 +1,59 @@
+# FAF in Docker container
+
+**Easiest way how to run FAF**
+---
+
+## How to deploy
+
+It is as simple as:
+
+`docker run --name faf -dit abrt/faf-image`
+
+However you also probably want to mount volumes to `/var/lib/postgres` and
+to `/var/spool/faf` not to loose database and FAF's data.
+
+`docker run --name faf -v /var/lib/faf-docker/faf:/var/spool/faf -v
+/var/lib/faf-docker/postgres:/var/lib/postgres/ -dit abrt/faf-image`
+
+If you run FAF for the first time, then there is no database. You have to
+initialize it.
+
+`docker exec faf init_db`
+
+Then FAF is ready for use.
+
+## What's next
+You can see incoming reports in webUI. It is accessible on `http://<container_IP>/faf`.
+
+Finding out container IP address:
+
+`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' faf`
+
+Also to send reports into your own FAF, you have to set up libreport on all
+machines, from with you wish to report into your own FAF. To do so, set up
+`URL = http://<container_IP>/faf` in `/etc/libreport/plugins/ureport.conf`.
+
+## Configuring FAF
+New containers comes with fully working and configured FAF (on top of basic configuration
+Fedora releases are added, caching is disabled and FAF accepts unknown components).
```

Comment:
comma after 'disabled' - It appears that you are missing a comma before the coordinating conjunction and in a compound sentence.